edx_proctoring/admin.py

```diff
@@ -334,6 +342,254 @@ def save_model(self, request, review, form, change):
         return
 
 
+class ExamAttemptFilterByStatus(admin.SimpleListFilter):
+    """
+    Quick filter to allow admins to see attempts by "status"
+    """
+
+    title = _('Status')
+    parameter_name = 'status'
+
+    def lookups(self, request, model_admin):
+        """
+        List of values to allow admin to select
+        """
+        return (
+            (ProctoredExamStudentAttemptStatus.created, _('Created')),
+            (ProctoredExamStudentAttemptStatus.submitted, _('Submitted')),
+        )
+
+    def queryset(self, request, queryset):
+        """
+        Return the filtered queryset
+        """
+
+        if self.value() in [ProctoredExamStudentAttemptStatus.created, ProctoredExamStudentAttemptStatus.submitted]:
+            return queryset.filter(status=self.value())
+        else:
+            return queryset
+
+
+class ExamAttemptFilterIsProctored(admin.SimpleListFilter):
```

Comment:
Can the BooleanFilter (http://django-filter.readthedocs.org/en/latest/ref/filters.html#booleanfilter) work for this one?